SonarQube.Client/SonarQubeService.cs

```diff
@@ -234,9 +234,12 @@ public Uri GetProjectDashboardUrl(string projectKey)
         {
             EnsureIsConnected();
 
-            const string ProjectDashboardRelativeUrl = "dashboard/index/{0}";
+            const string SonarQube_ProjectDashboardRelativeUrl = "dashboard/index/{0}";
```

Comment:
This is what we do for hotspots as well.

I originally created a request for this (see https://github.com/SonarSource/sonarqube-webclient-dotnet/commit/d2a8df353c0e3433091b21f4b922c57042d6874f) but it was messy integrating it into SLVS -- this code is used by the WPF commands and using Task.FromResult didn't feel right when looking at all these changes combined.

Reply:
FYI, SQ's URL was also changed, to `dashboard?id={0}` but there is a redirect so the old one still works. I left it as is for backwards compatibility with older SQ versions.

Reply:
That seems like a more sensible approach...